site/content/docs/developer/quick-start-build.md

```diff
@@ -0,0 +1,146 @@
+---
+title: "A Quick Start Guide to Building ZAP"
+description: "This guide will get you started with running ZAP and installing add-ons from source."
+images:
+- https://www.zaproxy.org/img/docs/developer/zap-dev-build.png
+tags: 
+- guide
+- tutorial
+type: page
+date: "2020-12-04"
+addSocialPreview: true
+---
+
+Have you ever wanted to play around with the ZAP codebase but felt it was too overwhelming? This guide will walk you through building ZAP from the command line regardless of the operating system and IDE you're using. Let’s jump right into it!
+
+## Preparation
+All you need is a working local installation of [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) and [JDK 8+](https://adoptopenjdk.net/).
+
+## Clone the Repo Locally
+Create a new directory on your system and run the following command in that directory to clone the core ZAP repository.
+```bash
+git clone https://github.com/zaproxy/zaproxy.git
+```
+
+## Running ZAP from Source
+Navigate to the cloned *zaproxy* folder and run the following command (if you are a Windows user, omit the `./`):
```

Comment:
On windows wouldn't you have to use `gradlew.bat`?

Reply:
I think it will work without, but dont have a windows box to hand to check right now ;)